### PR TITLE
test: Verify platform metrics are available

### PR DESCRIPTION
**Pull request by smarterclayton**
Ensure there are no regressions.

### CODE REVIEW

File: test/extended/prometheus/prometheus.go [hunk 1]
```diff
@@ -173,6 +173,29 @@ var _ = g.Describe("[Feature:Prometheus][Conformance] Prometheus", func() {
...
+			execPod := exutil.CreateCentosExecPodOrFail(oc.AdminKubeClient(), ns, "execpod", nil)
+			defer func() { oc.AdminKubeClient().CoreV1().Pods(ns).Delete(execPod.Name, metav1.NewDeleteOptions(1)) }()
+
+			tests := map[string]bool{
+				// track infrastructure type
+				`cluster_infrastructure_provider{type!=""}`: true,
+				`cluster_feature_set`:                       true,
+
+				// track installer type
+				`cluster_installer{type!="",invoker!=""}`: true,
```
Comment (LiliC) [inline test/extended/prometheus/prometheus.go]: How come we do not check that this value is > 0 as well?

Comment (smarterclayton) [reply test/extended/prometheus/prometheus.go]: good point

**Comment by s-urbaniak** [comment]
just one little concern from my side: Is this asserting a functioning monitoring stack? If not I suggest to create a dedicated `platforms` e2e test folder.

**Comment by smarterclayton** [comment]
Openshift e2e requires a cluster monitoring stack (non optional part), unless you mean something different?

**Comment by smarterclayton** [comment]
/retest

With Michal’s fixes

**Comment by smarterclayton** [comment]
To catch regressions

**Pull request merged**